### PR TITLE
Improve documentation for some functions

### PR DESCRIPTION
**Pull request by gassmoeller**

### CODE REVIEW

File: include/aspect/material_model/melt_global.h [hunk 1]
```diff
@@ -60,6 +60,14 @@ namespace aspect
         virtual void evaluate(const typename Interface<dim>::MaterialModelInputs &in,
                               typename Interface<dim>::MaterialModelOutputs &out) const;
 
+        /**
+         * Compute the equilibrium melt fractions for the given input conditions.
+         * @p in and @p melt_fractions need to have the same size.
+         *
+         * @p in Object that contains the current conditions.
```
Comment (tjhei) [inline include/aspect/material_model/melt_global.h]: ``@param in``

File: include/aspect/material_model/melt_global.h [hunk 2]
```diff
@@ -60,6 +60,14 @@ namespace aspect
...
+         * @p melt_fractions Vector of doubles that is filled with the
```
Comment (tjhei) [inline include/aspect/material_model/melt_global.h]: ``@param melt_fractions``

File: include/aspect/melt.h [hunk 3]
```diff
@@ -96,6 +96,14 @@ namespace aspect
     class MeltFractionModel
     {
       public:
+        /**
+         * Compute the equilibrium melt fractions for the given input conditions.
+         * @p in and @p melt_fractions need to have the same size.
+         *
+         * @p in Object that contains the current conditions.
```
Comment (tjhei) [inline include/aspect/melt.h]: same as above.

**Comment by gassmoeller** [comment]
Done

**Pull request merged**